src/aof.c

```diff
@@ -1432,7 +1432,9 @@ int loadSingleAppendOnlyFile(char *filename) {
             else
                 serverLog(LL_WARNING, "Error reading the RDB base file %s, AOF loading aborted", filename);
 
-            goto readerr;
+            if (feof(fp)) goto uxeof;
```

Comment:
I actually don't think we want that (i think we always want to got to cleanup with AOF_FAILED.
the `uxeof` part attempts to truncate the file and such, but this is an rdb failure, so i think it was wrong.
i think this code was only reachable on an incomplete RDB, which should never really happened (renamed after saving completed successfully).

bottom line, i think this one line should be dropped.

Reply:
yes, i agree, when I was preparing for this PR, I was tangled here...
I wanted to drop `uxeof` directly, i'm afraid I'll miss something, so I try to keep it consistent with the original code
i dropped it